src/main/java/com/example/onlinebookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -36,10 +36,19 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
 
     @ExceptionHandler(Exception.class)
     public ResponseEntity<Object> handleAllOtherExceptions(Exception e) {
+        return getResponseEntity(e.getMessage());
+    }
+
+    @ExceptionHandler(RuntimeException.class)
+    public ResponseEntity<Object> handleAllOtherRuntimeExceptions(RuntimeException e) {
+        return getResponseEntity(e.getMessage());
+    }
+
+    private ResponseEntity<Object> getResponseEntity(String message) {
         Map<String, Object> responseBody = new LinkedHashMap<>();
         responseBody.put("timespan", LocalDateTime.now());
         responseBody.put("status", HttpStatus.BAD_REQUEST);
```

Comment:
EntityNotFoundException should lead to NOT_FOUND status
EntityAlreadyPresentException should lead to CONFLICT
probably there are also other exceptions in project that may require some unique status

Reply:
fixed